astropy/utils/data.py

```diff
@@ -190,6 +190,12 @@ def get_readable_fileobj(name_or_obj, encoding=None, cache=False,
         long waits for a primary server that is known to be inaccessible
         at the moment.
 
+    http_headers : dict or None
+        HTTP request headers to pass into ``urlopen`` if needed. (These headers
+        are ignored if the protocol for the ``name_or_obj``/``sources`` entry
+        is not a remote HTTP URL.) In the default case (None), the headers are
+        "User-Agent: astropy" and "Accept: */*".
```

Comment:
I am mixing up RST with MarkDown... Does `*` mean anything special in rendering? Does this docstring need to be a raw string? Please check `giles`.

Reply:
Good point.  Switched to just using ``` `` ```literals